myuw/migrations/0009_resourcecategorypin.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
could you merge the three migration files into one?
just remove them and run makemigrations once.